### ISSUE TITLE
Fix current accessibility failings

### ISSUE THREAD

**Issue by Nurou** [Devops-ohtuprojekti/DevOpsCSAOS#216]
#227
- [x] Add missing HTML document `lang` attribute
- [x] Restructure HTML to only have one `main` element
- [x] Check structure of heading html tags (h1, h2, ...)

Estimate: 0h
